package.json

```diff
@@ -17,19 +17,22 @@
     "testEnvironment": "./test-environment.js"
   },
   "scripts": {
-    "clean": "rm -Rf node_modules ./modules/*/node_modules ./modules/*/dist",
+    "clean": "rm -Rf node_modules ./modules/*/node_modules ./modules/*/dist ./modules/*/.turbo",
     "lint": "tsdx lint modules/*/src",
     "build": "yarn turbo run build",
-    "test": "TESTING=true tsdx test --verbose --runInBand",
+    "test": "yarn workspaces run test",
```

Comment:
Does this preserve all the original parameters? Does this have the very same impact?

Reply:
yes works the same